dependencies.json

```diff
@@ -1,13 +1,13 @@
 {
-    "version_warpx": "25.11",
+    "version_warpx": "25.12",
     "version_amrex": "25.11",
     "version_pyamrex": "25.11",
     "version_picsar": "25.06",
     "version_pybind11": "v3.0.1",
     "version_picmi": "0.34.0",
-    "commit_amrex": "c4fcda1fb2ea238c3e6fa273dc9cfcde54b4cc51",
-    "commit_pyamrex": "c7c6e455a585ca43853009fb8df6fdce9e0a65cd",
-    "commit_picsar": "0c329e66010267662a82219f7de7abbd231463f4",
+    "commit_amrex": "25.12",
+    "commit_pyamrex": "25.12",
+    "commit_picsar": "25.06",
```

Comment:
@EZoni what would be a good logic that I did manually for the weekly commits:
If in fact the latest commit is described by the latest tag, I kept using the tag in `commit_*` even for weekly updates.

That avoids flip-flopping this line for dependencies that have releases less often.
Also makes it very obvious when a dependency of ours needs a new tag during `development` monthly cycles.

Reply:
That would be awesome, thank you! :)